cmd/admin-config-setkeys.go

```diff
@@ -0,0 +1,157 @@
+            }'
+`,
+}
+
+// configSetKeysMessage container to hold locks information.
+type configSetKeysMessage struct {
+	Status                   string `json:"status"`
+	setKeysConfigStatus      bool
+	SetKeysConfigNodeSummary []madmin.NodeSummary `json:"nodesSummary"`
+}
+
+// String colorized service status message.
+func (u configSetKeysMessage) String() (msg string) {
+	// For each node, print if set config was successful or not
+	for _, s := range u.SetKeysConfigNodeSummary {
+		if s.ErrSet {
+			// An error occurred when setting the new config
+			msg += console.Colorize("SetKeysConfigFailure",
+				fmt.Sprintf("Failed to apply the new configuration to `%s` (%s).", s.Name, s.ErrMsg))
+		} else {
+			msg += console.Colorize("SetKeysConfigSuccess",
+				fmt.Sprintf("New configuration applied to `%s` successfully.", s.Name))
+		}
+		msg += "\n"
+	}
+	// Print the general set config status
+	if u.setKeysConfigStatus {
+		msg += console.Colorize("SetKeysConfigSuccess",
+			"Setting new Minio configuration file has been successful.\n")
+	} else {
+		msg += console.Colorize("SetKeysConfigFailure",
+			"Setting new Minio configuration file has failed.\n")
+	}
+	return
+}
+
+// JSON jsonified service status Message message.
+func (u configSetKeysMessage) JSON() string {
+	if u.setKeysConfigStatus {
+		u.Status = "success"
+	} else {
+		u.Status = "error"
+	}
+	statusJSONBytes, e := json.Marshal(u)
+	fatalIf(probe.NewError(e), "Unable to marshal into JSON.")
+
+	return string(statusJSONBytes)
+}
+
+// checkAdminConfigSetKeysSyntax - validate all the passed arguments
+func checkAdminConfigSetKeysSyntax(ctx *cli.Context) {
+	if len(ctx.Args()) == 0 {
+		cli.ShowCommandHelpAndExit(ctx, "set", 1) // last argument is exit code
+	}
+}
+
+// main config setkeys function
+func mainAdminConfigSetKeys(ctx *cli.Context) error {
+
+	// Check command arguments
+	checkAdminConfigSetKeysSyntax(ctx)
+
+	// Set color preference of command outputs
+	console.SetColor("SetKeysConfigSuccess", color.New(color.FgGreen, color.Bold))
+	console.SetColor("SetKeysConfigFailure", color.New(color.FgRed, color.Bold))
+
+	// Get the alias parameter from cli
+	allArgs := ctx.Args()
+	if len(allArgs) < 2 {
+		fatalIf(probe.NewError(errors.New(
+			"Usage: mc admin config setkeys alias key1=value1 [key2=value2 key3.key4=value4 ...]")), "1- Invalid number of arguments\n")
+	}
+
+	aliasedURL := allArgs.Get(0)
+	argsMap := make(map[string]string)
+	for _, arg := range allArgs.Tail() {
+		argSplit := strings.SplitN(arg, "=", 2)
+		if strings.Index(arg, "{") == -1 && len(argSplit)%2 == 1 {
+			fatalIf(probe.NewError(errors.New(
+				"Usage: mc admin config setkeys alias key1=value1 [key2=value2 key3.key4=value4 ...]")), "2- Invalid number of arguments\n")
+		}
+		argsMap[argSplit[0]] = argSplit[1]
+	}
+
+	// Create a new Minio Admin Client
+	client, err := newAdminClient(aliasedURL)
+	fatalIf(err, "Cannot get a configured admin connection.")
```

Comment:
Since we use admin client in our madmin package documents too, we should use client in place connection in the above error message to be consistent.

Reply:
You meant `"Cannot get a configured admin client."`, right?
There are total  7 files with the same issue. I'll fix them all.